### PR TITLE
New package: XlightFTP.XlightFTP version 3.9.4.1

### LINKED ISSUE #148519: [Package Request]: Xlight FTP Server
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.xlightftpd.com/download.htm
Publisher: xlightftpd.com
Package Name: Xlight FTP Server
Description: FTP and SFTP server to make file transfer secure and easy to use.
Package Version: 3.9.4
Installer URL: https://www.xlightftpd.com/download/setup64.msi

### PR DESCRIPTION
**Pull request by Exorcism0666**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---
- Resolve #148519

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installer Verification Analysis Context Information: 
 Installer https://www.xlightftpd.com/download/setup64.msi
 Scan request id: '8e185cb0-3b51-4082-99b7-3080f1d6a27f'
 
 Installer https://www.xlightftpd.com/download/xlight.zip
 Scan request id: '06148e20-16de-4dfc-8e60-b5bcb45f17c7'
 
 Installer https://www.xlightftpd.com/download/xlight-x64.zip
 Scan request id: 'f9b11ab3-e590-459a-91c6-a13aab5c2521'
 
 Installer https://www.xlightftpd.com/download/setup32.msi

(Automated response - build 876.)

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installer Verification Analysis Context Information: 
 Installer https://www.xlightftpd.com/download/setup64.msi
 Scan request id: '8e185cb0-3b51-4082-99b7-3080f1d6a27f'
 
 Installer https://www.xlightftpd.com/download/xlight.zip
 Scan request id: '06148e20-16de-4dfc-8e60-b5bcb45f17c7'
 
 Installer https://www.xlightftpd.com/download/xlight-x64.zip
 Scan request id: 'f9b11ab3-e590-459a-91c6-a13aab5c2521'
 
 Installer https://www.xlightftpd.com/download/setup32.msi

(Automated response - build 876.)

File: manifests/x/XlightFTP/XlightFTP/3.9.4.1/XlightFTP.XlightFTP.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  InstallerSha256: D783C9B6E14E4434382C03B43685538E9CD8C6E57C4DBB23431465D90BB22627
+  ProductCode: '{5E04D858-A8C6-3CFD-D744-3754111BC0B7}'
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://www.xlightftpd.com/download/setup-x64.exe
+  InstallerSha256: E6F6A32C1C80B24B07CDD4DAED02C4AAA6CFF0A895D80700481EA693A5410DA7
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://www.xlightftpd.com/download/setup.exe
+  InstallerSha256: 94D907196A945737C25DE43AE5BBFC10719ED2EB2817AD7BED2096B937920561
```
Comment (stephengillie) [inline manifests/x/XlightFTP/XlightFTP/3.9.4.1/XlightFTP.XlightFTP.installer.yaml:26]:
```suggestion
  InstallerSha256: F1918D2F40CAC63E8179F56B214BF20174A3E22B610B9365D3326B2E6BBD0EEA
```

File: manifests/x/XlightFTP/XlightFTP/3.9.4.1/XlightFTP.XlightFTP.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://www.xlightftpd.com/download/setup32.msi
+  InstallerSha256: D783C9B6E14E4434382C03B43685538E9CD8C6E57C4DBB23431465D90BB22627
+  ProductCode: '{5E04D858-A8C6-3CFD-D744-3754111BC0B7}'
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://www.xlightftpd.com/download/setup-x64.exe
+  InstallerSha256: E6F6A32C1C80B24B07CDD4DAED02C4AAA6CFF0A895D80700481EA693A5410DA7
```
Comment (stephengillie) [inline manifests/x/XlightFTP/XlightFTP/3.9.4.1/XlightFTP.XlightFTP.installer.yaml:22]:
```suggestion
  InstallerSha256: F8F862020742CBAC045E20095A88E6A6125F7AF0E12066D9CCE18B14D3A563C7
```

**Review by stephengillie — CHANGES_REQUESTED** [review]

File: manifests/x/XlightFTP/XlightFTP/3.9.4.1/XlightFTP.XlightFTP.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://www.xlightftpd.com/download/setup64.msi
+  InstallerSha256: 4304448292D7B63CE849EF7BDE83DA160649B6D7CA9B8C905B0E7697451307E0
+  ProductCode: '{5E04D858-A8C6-3CFD-D744-3754111BC0B7}'
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://www.xlightftpd.com/download/setup32.msi
```
Comment (Trenly) [inline manifests/x/XlightFTP/XlightFTP/3.9.4.1/XlightFTP.XlightFTP.installer.yaml:16]: On Win11 this installer fails a virus check? @stephengillie

image

Comment (stephengillie) [reply manifests/x/XlightFTP/XlightFTP/3.9.4.1/XlightFTP.XlightFTP.installer.yaml:16]: Do you have the Security Intelligence Version of Defender that caught this?

Comment (Trenly) [reply manifests/x/XlightFTP/XlightFTP/3.9.4.1/XlightFTP.XlightFTP.installer.yaml:16]: image

Comment (Trenly) [reply manifests/x/XlightFTP/XlightFTP/3.9.4.1/XlightFTP.XlightFTP.installer.yaml:16]: Might be a false positive, but not sure

**Comment by stephengillie** [comment]
Verify: 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**